be/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+services:
+  api:
+    image: node:18-alpine
+    command: sh -c "npm install && npm run dev"
+    working_dir: /server
+    volumes:
+      - .:/server
+    ports:
+      - 4000:4000 
+    environment:
+      PORT: 4000
+  mongo:
+    image: "mongo"
+    volumes:
+      - mongodb:/data/db
+
+volumes:
+  mongodb:
```

Comment:
seems incomplete, leftover?

Reply:
I believe this is needed, it instructs docker compose to create a docker volume if it does not already exist.